pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```diff
@@ -48,9 +61,13 @@ stdenv.mkDerivation (finalAttrs: {
   buildPhase = ''
     runHook preBuild
 
+    # darwin needs a writable electron directory
+    cp -r ${electronDist} electron-dist
+    chmod -R u+w electron-dist
+
```

Comment:
This too.

Reply:
If we decide we want to sacrifice some simplicity to lower disk space usage on Linux, then we should changed it.
Until then, I'd rather stay with the current, simpler logic.

Reply:
I am not sure here. Nix store is read and bind mounted on Linux, so apart from during build phase, it wouldn't even be writable, even by current user.

If it's not needed on Linux, we should not do it.